docs/src/docs/asciidoc/guides/cassandra.adoc

```diff
@@ -37,6 +37,13 @@ Spring Session, which is in turn backed by Cassandra.
 When a Cassandra cluster is available, the primary configuration properties are available via
 `@EnableCassandraHttpSession`
 
+=== Application context provided dependencies.
+
+In order to enable Spring session's session management capabilities using Cassandra it is mandatory for your application
+to configure and auto-wire a bean of the type `com.datastax.driver.core.Session` which must define a cluster contact
+point host, as well as a port number. When injected, this bean will be used in conjunction with
+`org.springframework.cassandra.core.CqlOperations` to issue queries against the Cassandra cluster.
+
```

Comment:
This is a bit out of date now. spring-session doesn't do anything with the host/contact points, it just needs a `CqlOperations` bean.